tests/Core/Tokenizer/BackfillNumericSeparatorTest.inc

```diff
@@ -34,6 +34,9 @@ $foo = 0b0101_1111;
 /* testOctal */
 $foo = 0137_041;
 
+/* testExplicitOctal (Introduced in PHP 8.1) */
```

Comment:
No need to include the text `(Introduced in PHP 8.1)` in the test comment - it's clearer to read without it